src/Python-Windows/chevahbs

```diff
@@ -9,58 +9,29 @@
 source ../../functions.sh
 source ../../functions_pythia.sh
 
-# The usual two arguments, e.g. "Python" and "2.7.18",
+# The usual two arguments, e.g. "Python" and "3.9.0",
 # but the installation path is also needed, because this script is special.
 NAME=$1
 VER=$2
 INSTALL_DIR=$3
 
-# Construct the relevant download link for the MSI installer.
-EXT="msi"
-if [ "$ARCH" = "x64" ]; then
-    # Upstream installer filename ends in ".msi" or ".amd64.msi".
-    EXT="amd64.msi"
-fi
-INSTALLER="python-${VER}.${EXT}"
-LINK=https://www.python.org/ftp/python/"$VER"/"$INSTALLER"
-
-WIPER="https://binary.chevah.com/production/sxs-manifest-wiper/manifest-wiper.exe"
-
-# Get required files locally, no need to unpack in build/ for msi/exe files.
-download_sources $NAME $VER $LINK $EXT
-download_sources manifest wiper $WIPER exe
-
-# Get Windows redists locally, to pick them up when installing.
-echo "### Collecting Windows redistributables... ###"
-./get_latest_redist.sh
+# There's no actual building on Windows, no headers and no built libs.
+rmdir "$INSTALL_DIR"/{include,lib}
 
-DEST_DIR=$INSTALL_DIR/lib
-ARCH_DIR=$ARCH
-if [ "$ARCH" = "x64" ]; then
-    # VC redist x64 DLLs actually use "amd64" in filenames.
-    ARCH_DIR="amd64"
+# Construct the relevant download link for the Windows version.
+EXT="amd64.zip"
+if [ "$ARCH" = "x86" ]; then
+    EXT="win32.zip"
 fi
+ARCHIVE="python-${VER}.${EXT}"
+LINK="https://www.python.org/ftp/python/${VER}/python-${VER}-embed-${EXT}"
```

Comment:
Great move here.
I hope the embed version should remove the need work manifest wiper...as the files are already "'wiped"